src/ClassFile/types/fields.ts

```diff
@@ -1,10 +1,11 @@
-import { AttributeType } from './attributes';
+import { AttributeInfo } from './attributes';
 
-export interface FieldType {
+export interface FieldInfo {
   accessFlags: number;
   nameIndex: number;
   descriptorIndex: number;
-  attributes: Array<AttributeType>;
+  attributesCount: number;
```

Comment:
For the count fields, I originally removed them since we could just use `array.length`. It is fine that we are sticking to the classfile representation though